### PR TITLE
3.1.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Added release notes
    - Removed PBs marked deprecated from contents
    - Added filtering records section
    - Added information about voice support in setup, usage, and readme

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -62,19 +62,29 @@ The following permission allow a user to interact with the **FortiSOAR AI Assist
...
 The **FortiAI** solution depends on the following connectors and widgets:
 
 - **AI Assistant Utils** connector to help interact with LLMs like OpenAI.
 
 >[!NOTE]
->The AI Assistant Utils connector does not require a configuration. For more details, see the [AI Assistant Utils Connector](https://docs.fortinet.com/document/fortisoar/1.0.0/ai-assistant-utils/690/ai-assistant-utils-v1-0-0) document.
+>The AI Assistant Utils connector does not require a configuration. For more details, see the [AI Assistant Utils Connector](https://docs.fortinet.com/fortisoar/connectors/ai-assitant-utils) document.
 
-- The **AI Assistant** widget to add the **FortiSOAR AI Assistant** bot to the FortiSOAR user interface. For more details, see the [AI Assistant Widget](https://docs.fortinet.com/document/fortisoar/1.0.0/ai-assistant-utils/690/ai-assistant-utils-v1-0-0) document.
+- The **AI Assistant** widget to add the **FortiSOAR AI Assistant** bot to the FortiSOAR user interface. For more details, see the [AI Assistant Widget](https://docs.fortinet.com/fortisoar/connectors/ai-assitant-utils) document.
```
Comment (cs-ankit-gupta) [inline docs/setup.md]: To AI Assistant widget doc link

File: release_notes.md [hunk 2]
```diff
@@ -1,59 +1,10 @@
 # What's New
 
-## Support for FortiAI GenAI Assistant
+## Support for Voice Recognition in AI Assistant
```
Comment (cs-ankit-gupta) [inline release_notes.md]: New release note as that now we are not deleting the thread from OpenAI, we are only clearing the conversation on clicking the 'Clear Conversation' from bot

**Review by cs-ankit-gupta — CHANGES_REQUESTED** [review]

File: docs/setup.md (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply docs/setup.md]: @cs-abhishek-shukla

File: release_notes.md (hunk 2 above)
Comment (cs-suyog-jinturkar) [reply release_notes.md]: @cs-abhishek-shukla

File: docs/prompts-for-filtering-records.md
Comment (cs-suyog-jinturkar) [inline docs/prompts-for-filtering-records.md:1]: LGTM. Please add remaining ones

File: docs/advanced-usage.md
Comment (cs-suyog-jinturkar) [inline docs/advanced-usage.md:1]: link for filtering records prompts needs to be added here in its respective section.

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Pointing to Dev repo as the branch is merged in develop after release, ok? @cs-suyog-jinturkar @cs-ankit-gupta

File: release_notes.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: same thread? not new thread? @cs-ankit-gupta

File: docs/prompts-for-filtering-alert-records.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Verify that the results include only alerts generated within the specified date range, ensuring no alerts outside the period are displayed.
+
+### Other examples
+
+Similarly, you can use the following prompts to filter alert records.
+
+1. Give me alerts assigned to <user-1>.
+ 
+2. Give me all alerts generated between October 1, 2024, and October 15, 2024
```
Comment (cs-rahul-vishwakarma) [inline docs/prompts-for-filtering-alert-records.md]: Please change to 
`Give me all alerts created between October 1, 2024, and October 15, 2024`

File: docs/prompts-for-filtering-alert-records.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+ 
+3. Show me all alerts where the *severity* is either high or critical, the status is open or in progress, and the type is malware or phishing. The alerts should have been created on within the last 7 days, and the source IP should start with '192.168.1.
+ 
+4. List all in-progress alerts, excluding those of the phishing or suspicious type.
+
```
Comment (cs-rahul-vishwakarma) [inline docs/prompts-for-filtering-alert-records.md:99]: Add more prompts:
`filter out all the alerts with technique ID T1074.001`
`Find alerts triggered by the rule PH_RULE_UEBA_AI_FILE_WRITTEN`

File: docs/prompts-for-filtering-incident-records.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+![Incidents with ID greater than 400 filtered](./res/filter-incidents-id-greater-400.png)
+
+### Review Results
+
+Verify that the results include only incidents with an ID greater than 400, ensuring no incidents with a lower ID are included.
+
+### Other examples
+
+Similarly, you can use the following prompts to filter incident records.
```
Comment (cs-rahul-vishwakarma) [inline docs/prompts-for-filtering-incident-records.md:26]: Replace the prompts with following examples:
`Give me incidents led by me.`
`Give me incidents created in last 7 days.`
`Give me incidents tagged as super, and not led by <user-1>`
`fetch incidents of type phishing which are in containment phase`

File: docs/prompts-for-filtering-indicator-records.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+* **Prompt**:  
+   > _"Give me all indicators of type URL."_
+
+* **Expected Outcome**:  
+   Lists all indicators of *type* **URL**.
+
+### Review Results
+
+Verify that the results include only incidents with an ID greater than 400, ensuring no incidents with a lower ID are included.
```
Comment (cs-rahul-vishwakarma) [inline docs/prompts-for-filtering-indicator-records.md]: Please update this review result to followiing:
`Verify that the results include only indicators of type URL are filtered`

File: docs/res/filter-alert-assigned-self.png
Comment (cs-rahul-vishwakarma) [inline docs/res/filter-alert-assigned-self.png:1]: Sir, we need to change all the images, as applied filters are not visible. (It's a known issue in 7.6.1)

File: release_notes.md [hunk 7]
```diff
@@ -1,59 +1,16 @@
...
-  - Get Incident Response Plan
-  - Get Alert Report
-  - Get Incident Report
-  - Get Alert MITRE Insight
-  - Get Incident MITRE Insight
-
->[!Note]
->Delete these playbooks manually when upgrading from an older **FortiAI** version.
\ No newline at end of file
+Now, threads are retained within OpenAI, enabling improved historical reference and continuity. Each new prompt still initiates a fresh thread, while the user interface continues to clear conversation history after each prompt, ensuring a clean workspace.
```
Comment (cs-rahul-vishwakarma) [inline release_notes.md]: This sounds confusing! Need to rephrase this.

File: README.md [hunk 8]
```diff
@@ -15,6 +15,16 @@ FortiAI is a context-aware GenAI security assistant that transforms how security
 
 Its unique feature, among others, is its ability to generate playbook blocks tailored to scenarios defined by you, giving you a head start in automating and enhancing your use cases. Whether you are a seasoned playbook developer or just starting out, FortiAI is your go-to partner for streamlined and efficient playbook design.
 
+## Voice Support Overview
+
+FortiAI now includes voice-to-natural-language functionality, enabling hands-free interaction. This feature allows users to give voice commands for tasks such as generating playbooks, filtering records, or blocking indicators, streamlining workflows and increasing accessibility. This feature has multiple benefits like the following:
```
Comment (cs-rahul-vishwakarma) [inline README.md:27]: It's a **Tap-and-Hold** feature so saying "hands-free interaction" seems counterintuitive.

**Review by cs-rahul-vishwakarma — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (cs-suyog-jinturkar) [reply README.md:27]: let it be.. :)

File: docs/prompts-for-filtering-alert-records.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply docs/prompts-for-filtering-alert-records.md]: Done

File: docs/prompts-for-filtering-alert-records.md (hunk 4 above)
Comment (cs-abhishek-shukla) [reply docs/prompts-for-filtering-alert-records.md:99]: Done

File: docs/prompts-for-filtering-incident-records.md (hunk 5 above)
Comment (cs-abhishek-shukla) [reply docs/prompts-for-filtering-incident-records.md:26]: Done

File: docs/prompts-for-filtering-indicator-records.md (hunk 6 above)
Comment (cs-abhishek-shukla) [reply docs/prompts-for-filtering-indicator-records.md]: Done

File: release_notes.md (hunk 7 above)
Comment (cs-abhishek-shukla) [reply release_notes.md]: Changed

**Pull request merged**